packages/deb/hazelcast/DEBIAN/control

```diff
@@ -5,7 +5,6 @@ Section: imdg
 Priority: optional
 Architecture: all
 Conflicts: ${CONFLICTS}
-Depends: java21-sdk-headless | default-jdk-headless
```

Comment:
Why this change? This will hurt UX as users will have to install java on their own

Reply:
This breaks backward compatbility. It will break customer automation that assumes our package will install everything. Sdkman is popular on dev machines, not on production ones. I think we don't support RPI. 

Alos, It's a common practice in deb to not force users to install dependencies manually

Reply:
ok, turned out that RasPi OS had issues, because the default Java package we have now was *not* present in 5.5 packages. I totally misunderstood the issue, thanks @ldziedziul for checking that out! 

Instead of removing the package data, I've added slightly different execution, where user can point to snapshot repository using the script.